lib/perf_hooks.js

```diff
@@ -166,50 +166,89 @@ function getMilestoneTimestamp(milestoneIdx) {
 }
 
 class PerformanceNodeTiming extends PerformanceEntry {
-  get name() {
-    return 'node';
-  }
+  constructor() {
+    super();
 
-  get entryType() {
-    return 'node';
-  }
+    ObjectDefineProperties(this, {
+      name: {
+        enumerable: true,
+        get() {
+          return 'node';
+        }
+      },
```

Comment:
We might as well make these plain properties in this case, right? Providing getters this way results in a major perf hit, see e.g. https://github.com/nodejs/node/pull/34459

Reply:
IMHO this creates a new function only when a new `nodeTiming` object is created (which is only once) or am I missing something? Please, don't tell me this has the effect of reevaluating at each call because I have lots of respect for V8?

Reply:
Oh, you mean the static properties, yes, sure

Reply:
Oh, right, yeah, that’s fine then :+1: